routers/web/repo/migrate.go

```diff
@@ -128,7 +129,7 @@ func handleMigrateRemoteAddrError(ctx *context.Context, err error, tpl base.TplN
 		case addrErr.IsProtocolInvalid:
 			ctx.RenderWithErr(ctx.Tr("repo.mirror_address_protocol_invalid"), tpl, form)
 		case addrErr.IsURLError:
-			ctx.RenderWithErr(ctx.Tr("form.url_error"), tpl, form)
+			ctx.RenderWithErr(fmt.Sprintf("%q%s", addrErr.Host, ctx.Tr("form.url_error")), tpl, form)
```

Comment:
You really should do:

```suggestion
			ctx.RenderWithErr(ctx.Tr("form.url_error", addrErr.Host), tpl, form)
```

and fix the form.url_error string.